src/coreclr/inc/corinfo.h

```diff
@@ -1204,7 +1204,7 @@ enum CorInfoAccessAllowedHelperArgType
     CORINFO_HELPER_ARG_TYPE_Field   = 1,
     CORINFO_HELPER_ARG_TYPE_Method  = 2,
     CORINFO_HELPER_ARG_TYPE_Class   = 3,
-    CORINFO_HELPER_ARG_TYPE_Module  = 4,
+    // CORINFO_HELPER_ARG_TYPE_Module  = 4,
```

Comment:
Can we remove it and change the values if we are changing JIT-EE GUID anyway?

Reply:
Doesn't it need r2r bump then?